module/platform/darwin/HardwareIO/data/config/HardwareIO.yaml

```diff
@@ -1,7 +1,7 @@
 battery:
```

Comment:
I dont know about these values

Reply:
Most of these changes relate to the old default configuration file being for the darwin. The only values of interest here are the knee offset which was updated from `pi / 8` to `0.45` which we determined from `scripttuner` and zombie